### PR TITLE
RE-231 Update rpc_release and SHA's for r14.3.0

### PR DESCRIPTION
**Pull request by odyssey4me**
Issue: [RE-231](https://rpc-openstack.atlassian.net/browse/RE-231)

### CODE REVIEW

File: ansible-role-requirements.yml [hunk 1]
```diff
@@ -18,57 +18,3 @@
...
-- name: lxc_hosts
-  scm: git
-  src: https://git.openstack.org/openstack/openstack-ansible-lxc_hosts
-  version: 09a9ff6297038f4eba49c4cac31ca78f4abb9d51
-# TODO(odyssey4me)
-# Remove this once this SHA is included in our OSA SHA
-- name: rabbitmq_server
-  scm: git
-  src: https://git.openstack.org/openstack/openstack-ansible-rabbitmq_server
-  version: 6546786a28ad5e28a9158c1eb3912915b427e9f7
```
Comment (evrardjp) [inline ansible-role-requirements.yml]: As you will bump the artifacts, you'll also probably bump erlang/rabbitmq (to verify what we have defined in our apt artifact sources).

This also means we won't have the pin in b8f4fe98998268197782603f029ae1cb29e5b8c5.

Did you foresee that, or should I investigate further?

Comment (odyssey4me) [reply ansible-role-requirements.yml]: The erlang patches are not yet included in the integrated build for OSA, so we don't need any changes.

Comment (evrardjp) [reply ansible-role-requirements.yml]: Yes, I agree, but I guess they were included in the role's for a reason...

Comment (odyssey4me) [reply ansible-role-requirements.yml]: As you heard in the discussion yesterday (or was it the day before), there's some dispute about whether they're the right thing to use... so at this stage I'd rather let it play out. Also, the less tech debt we carry, the better.

**Comment by odyssey4me** [comment]
recheck_cit_trusty_deploy_swift - failed kibana tests, this is a flaky test so doing a recheck

minorupgrade/leapfrogupgrade tests failed due to lack of artifacts - only once the artifact/deploy tests pass can we build them: ``FAILED! => {"changed": false, "command": "/usr/bin/lxc-create --name LXC_NAME --quiet --template download --bdev dir -- --dist ubuntu --release xenial --arch amd64 --force-cache --server rpc-repo.rackspace.com --variant default-r14.3.0 --no-validate --keyserver hkp://p80.pool.sks-keyservers.net:80", "err": "", "failed": true, "msg": "Failed executing lxc-create.", "rc": 1}``

**Comment by odyssey4me** [comment]
recheck_cit_trusty_deploy_swift - failed allocating pubcloud node

**Comment by major** [comment]
recheck_cit_trusty_leapfrogupgrade_swift

**Comment by odyssey4me** [comment]
Artifacts and deploy jobs passed. The others will repeat fail until we build artifacts, so I'll kick off the job to do it now.

**Comment by odyssey4me** [comment]
Artifact build for r14.3.0 started: https://rpc.jenkins.cit.rackspace.net/view/Artifacts/job/RPC-Artifact-Build_newton-periodic/44/

**Comment by odyssey4me** [comment]
Artifact build complete.

recheck_cit_trusty_leapfrogupgrade_swift
recheck_cit_trusty_minorupgrade_swift
recheck_cit_xenial_minorupgrade_swift

**Pull request merged**